signer/universal/universal_signers.go

```diff
@@ -0,0 +1,9 @@
+// +build !pkcs11
+
+package universal
+
+// Without any build flags, the only local signer that should be
+// activated is the file-backed local signer.
+var localSignerList = []localSignerCheck{
+	fileBackedSigner,
+}
```

Comment:
Again, this split seems like overkill.  Instead:
1. Get rid of localSignerCheck
2. Push any checking logic in fileBackedSigner / pkcs11Signer into the local.NewSigner()FromFile / pkcs11.New().
3. Replace calls to localSignerCheck methods with just the constructors.

That way, when the build flags are off, the PKCS#11 stuff will just fail.